wizeline/remix-project-lab
# Feat/filter tags page

# What does this PR do?

This PR add all the filter-tags functionality:
- Add the Statuses, Labels and Innovation Tiers tab
- Add the column `stage` to the `projectStatus` table

# Where should the reviewer start?

The reviewers should set themselves as `ADMIN` in the `User` table and go to the manager page

# How should this be manually tested?

- Go to the manager page from the dropdown menu that is on the Header (email)
- The tab `Filter Tags` is selected by default, so play with the Labels, Statuses and Innovation Tiers functionalities
- You should be able to create, modify and delete any value from these tabs

# What are the relevant tickets?

#32 #33 #34 

# Screenshots

<img width="912" alt="image" src="https://user-images.githubusercontent.com/96010361/190709855-a187d53d-cd5c-4f19-9048-8fd080498354.png">

![image](https://user-images.githubusercontent.com/96010361/191517296-80d706a0-0904-4891-952d-c45d35875379.png)

<img width="988" alt="image" src="https://user-images.githubusercontent.com/96010361/190710020-1b9b747f-bb19-40f6-b89e-7d9b8fc8aeff.png">


# Checklist
